core/shared/src/main/scala/monocle/std/Validated.scala

```diff
@@ -0,0 +1,33 @@
+package monocle.std
+
+import monocle.{Prism, Iso, PIso, PPrism}
+
+import cats.data.Validated
+import cats.syntax.either._
+import cats.syntax.validated._
+
+object validation extends ValidatedOptics
```

Comment:
should the object be called validated as well?

Reply:
I agree